scripts/test-renode.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+SCRIPT_SRC=$(realpath ${BASH_SOURCE[0]})
+SCRIPT_DIR=$(dirname $SCRIPT_SRC)
+TOP_DIR=$(realpath $SCRIPT_DIR/..)
+if [[ "${BASH_SOURCE[0]}" != "${0}" ]]; then
+	echo "You must run this script, rather then try to source it."
+	echo "$SCRIPT_SRC"
+	exit 1
+fi
+
+source $SCRIPT_DIR/build-common.sh
+
+init
+
+case $CPU in
+	vexriscv)
+    		CPU_TYPE="VexRiscv"
+		;;
+	picorv32)
+    		CPU_TYPE="PicoRV32"
+		;;
+	*)
+		echo "Simulating the current configuration is not supported in Renode yet - skipping the test"
+		exit 0
+		;;
+esac
+
+if [ "$PLATFORM" == "mimas_a7" ] || [ "$PLATFORM" == "netv2" ] || [ "$PLATFORM" == "pano_logic_g2" ]; then
+    # the test load binaries into the flash to avoid using netboot on CI server
+    echo "$PLATFORM platform does not have flash memory - skipping the test"
+    exit 0
+fi
+
+if [ "$FIRMWARE" == "zephyr" ]; then
+    if [ "$PLATFORM" == "icebreaker" ]; then
+        # running Zephyr firmware directly from flash is not supported at the moment
+        # as it requires to enable XIP and include flash section in the DTS;
+        # TODO: add this in the future
+        echo "Running $FIRMWARE directly from flash is currently not supported - skipping the test"
+        exit 0
+    fi
+fi
+
+# Download prebuilt renode Release with new terminal tester support
+conda install -c antmicro/label/test -c conda-forge renode=v1.8.2_with_new_terminal_tester
```

Comment:
Right now it uses a special conda package (including an improved terminal tester that is not yet released on master). This will be replaced with the master-based package in the future.

Reply:
Okay, SGTM.